packages/perspective-viewer-datagrid/src/js/style_handlers/table_cell/row_header.js

```diff
@@ -20,7 +20,11 @@ export function cell_style_row_header(regularTable, td, metadata) {
         dx: 0,
         dy: metadata.y - metadata.y0 + 1,
     });
+
+    const is_last_item = !next || next.row_header === undefined;
```

Comment:
This only checks whether the row is the last row in the _viewport_, not the dataset. it just makes the last row visible on screen always appear folded.

https://github.com/user-attachments/assets/6da2eedb-a782-4ff6-a916-53fa35a56e4b